app/commands/mentor/request/accept_external.rb

```diff
@@ -21,6 +22,14 @@ def call
           )
         end
       end
+
+      private
+      def log_metric!(request)
+        Metric::Queue.(:request_mentoring, request.created_at, request:, track:, user:)
```

Comment:
I think this should be a separate metric "RequestPrivateMentoring"

```suggestion
        Metric::Queue.(:request_mentoring, request.created_at, request:, track:, user:)
```

Reply:
https://github.com/exercism/website/pull/2768/commits/6a160264967a3ce75494262a07f300c4c9553c9a

I'll need to run some queries to fixup the existing metrics.